README.rst

```diff
@@ -90,25 +90,59 @@ Test Remote Server
 
 Integration tests can be executed on a remote server.
 
+To run the integration test suite, first build the integration tests container
+
 .. code-block:: shell
 
     docker-compose build tests
 
+or download a pre-built container from `Dockerhub <https://hub.docker.com/r/mozilla/remote-settings-integration-tests>`_.
+
+Next run the tests, supplying config values as necessary. Config values are
+set as environment variables provided to the Docker container. See
+``tests/conftest.py`` for descriptions of all of the config options that are
+available.
+
+Note that the tests assume that the server has the ``attachments``,
+``changes``, ``history``, and ``signer`` plugins enabled. It may optionally
+have the ``email`` plugin installed.
+
+To have the tests bootstrap themselves (i.e. when ``SKIP_SERVER_SETUP=false``):
+
+- a user account should available with the ability to create users, buckets, and
+  collections
+- the account should also be able to assign users to groups
```

Comment:
This permission does not really exist in itself. It the user has the permission create/write buckets, then it can write groups

Reply:
[d57f3d5](https://github.com/mozilla/remote-settings/pull/227/commits/d57f3d50673d86a2da6792c7f0c9ec4085bc5056)